packages/lib/gpcircuits/package.json

```diff
@@ -40,6 +41,7 @@
     "@pcd/util": "^0.5.0",
     "@semaphore-protocol/identity": "^3.15.2",
     "@types/chai": "^4.3.5",
+    "@types/lodash": "^4.17.0",
```

Comment:
I still suspect you could get away with including `lodash` in devDependencies rather than `@types/lodash` here.  Let me know if that didn't work out.